### PR TITLE
Move current web to Next js 15 RC with React 19 RC and compiler

### PR DESCRIPTION
**Pull request by Carlltz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,14 @@
 FROM node:20-alpine3.20 AS build
```
Comment (Carlltz) [inline Dockerfile:1]: @foodelevator Does this dockerfile work or we need some special sauce?

Comment (foodelevator) [reply Dockerfile:1]: how de fak shud i knaw if it worken?

Comment (Carlltz) [reply Dockerfile:1]: I mean it workeding for me on my machine so fak it let's YOLO! :smiley:

Comment (foodelevator) [reply Dockerfile:1]: but nothing special is needed, so stuff should just work

**Pull request merged**